python/srf/_pysrf/src/ingress_port.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+* SPDX-FileCopyrightText: Copyright (c) 2021-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+* SPDX-License-Identifier: Apache-2.0
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include <pysrf/node.hpp>
+#include <pysrf/types.hpp>
+
+#include <srf/segment/ingress_port.hpp>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/pytypes.h>
+
+namespace srf::segment {
+namespace py = pybind11;
+
+template <>
```

Comment:
I'm struggling to find the difference between this and the definition in the .hpp file.

It feels like this could be replaced with:

```
template class IngressPort<pysrf::PyHolder>;
```

Reply:
The difference is in the RxNode constructor, its built using a PythonNode instead of directly with an RxNode.